packages/actions/src/index.js

```diff
@@ -1,8 +1,9 @@
 export { prefixedValueMirror as makeActionTypes } from '@redux-tools/utils';
 
-export { default as makeActionCreator } from './makeActionCreator';
-export { default as makeConstantActionCreator } from './makeConstantActionCreator';
-export { default as makeSimpleActionCreator } from './makeSimpleActionCreator';
+export { default as configureActionCreator } from './configureActionCreator';
+export { default as makePayloadMetaActionCreator } from './makePayloadMetaActionCreator';
```

Comment:
import empty, payload and payloadmeta here and export them as empty, constant, simple, payload (i.e. have all the possible reexports here)